manifests/deploy/openshift/olm/bundle/4.2/meteringoperator.v4.2.0.clusterserviceversion.yaml

```diff
@@ -193,7 +193,7 @@ metadata:
     categories: OpenShift Optional, Monitoring
     certified: "false"
     containerImage: quay.io/openshift/origin-metering-ansible-operator:latest
-    createdAt: 2019-01-01T11:59:59Z
+    createdAt: "2019-01-01T11:59:59Z"
```

Comment:
This is causing `make verify` to fail in CI. I'm guessing it's related to your `helm` version or something. Can you just manually change this back to avoid the issue for now?